### ISSUE TITLE
Package not on Stackage

### ISSUE THREAD

**Issue by ysangkok** [adinapoli/threads-supervisor#30]
Hi, thank you for this great package, I appreciate it.

I noticed the package isn't on Stackage (thread-supervisor is though, but that is a different package!)

[The instructions](https://github.com/commercialhaskell/stackage/blob/master/MAINTAINERS.md#adding-a-package) say that I should request it from you first before submitting it myself. So consider this my formal request for adding the package to Stackage.

Many regards!

**Comment by adinapoli** [comment]
Hey @ysangkok , thanks for your kind words and for checking out this package.

I am certainly not opposed to add this package to Stackage (in fact, my other package, `mandrill` is on Stackage or used to be prior to the `aeson` breaking changes). The only problem is that, while keeping a package on Stackage is mostly hassle-free, it introduces a certain overhead now and again when there are changes in the dependencies.

Unfortunately these days I have very little spare time to dedicate to maintaining my packages (unless it's directly driven from some client's need at work) and I fear that if I go down the Stackage route this would introduce additional overhead for me. Having said that, I will see what I can do.

In the meantime, cannot you simply add this package to the `extra-deps` section of your `stack.yaml`? I agree that having this on Stackage as a first-class package would be nicer, but doing the former should pretty much sort you out, as this package is very lightweight dependencywise anyway, so you wouldn't (hopefully) have any problem with building it as part of a bigger snapshot.

**Comment by ysangkok** [comment]
Yes, we currently have the package in `extra-deps`, and it is working fine. I am just trying to cut down on `extra-deps` so that it is more visible how the remaining ones are the ones that we can't use the snapshot version of. Because almost all packages are on Stackage, and it is even allowed to add packages you don't maintain, this goal seems within reach.

I saw that @Mikolaj uploaded the latest release. Would Mikolaj be interested in adding it to Stackage? I see he already has 7 packages on there.

**Comment by Mikolaj** [comment]
Hi! It would be unfair in the sense that if any problems arose, I'd come begging Alfredo for help, anyway. ;)

I don't use this package any more, so I doubt I'd ever be competent tending to it without harassing innocent bystanders.

**Comment by juhp** [comment]
Package is still in Nightly - I think this ticket can be closed
